src/plugins/arkeo/StakingCard.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from '@chakra-ui/react'
+import { DefiProvider } from 'features/defi/contexts/DefiManagerProvider/DefiCommon'
+import type { EarnOpportunityType } from 'features/defi/helpers/normalizeOpportunity'
+import { useTranslate } from 'react-polyglot'
+import { AssetIcon } from 'components/AssetIcon'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { ArkeoCard } from './ArkeoCard'
+
+type StakingCardProps = {
+  onClick: (arg: EarnOpportunityType) => void
+} & EarnOpportunityType
+
+export const StakingCard: React.FC<StakingCardProps> = props => {
+  const translate = useTranslate()
+  const { onClick, ...rest } = props
```

Comment:
If you go with passing down an id, that will be a non-problem but if you keep this implementation, rename `rest`
```suggestion
  const { onClick, ...earnOpportunity } = props
```